marker_lints/tests/ui/diag_msg_uppercase_start.stderr

```diff
@@ -1,16 +1,19 @@
-warning: this message starts with an uppercase character
-  --> $DIR/diag_msg_uppercase_start.rs:21:31
-   |
-21 |     cx.emit_lint(DUMMY, expr, "X <-- starting with upper case");
-   |                               ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
-   |
-   = note: `#[warn(marker::diag_msg_uppercase_start)]` on by default
+error[E0603]: module `expr` is private
+ --> $DIR/diag_msg_uppercase_start.rs:3:23
+  |
+3 | use marker_api::{ast::expr::ExprKind, context::MarkerContext};
```

Comment:
This doesn't look right

Reply:
The new ui_test version blesses tests by default. That's why I didn't see the fail. I don't like that, I'll make it error by default. There is a setting for that